spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -416,4 +420,12 @@ def then_I_see_an_unschedule_button
       expect(page).to have_css("button", text: "Unschedule publishing")
     end
   end
+
+  def and_I_click_the_unschedule_button
+    click_on 'Unschedule publishing'
+  end
+
+  def base_path
```

Comment:
This isn't a blocker, but I'm not sure of the benefit of splitting this out.
`stub_publishing_api_destroy_intent(base_path)` is not much shorter than `stub_publishing_api_destroy_intent('/how-to-be-the-amazing-1')`